concordium-node/src/configuration.rs

```diff
@@ -295,18 +302,34 @@ pub struct BakerConfig {
     pub account_cache_size: u32,
     #[structopt(
         long = "baker-credentials-file",
+        hidden = true,
         help = "Path to the baker credentials file. If the path is relative it is interpreted \
                 relative to the node process' working directory.",
         env = "CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE"
     )]
-    pub baker_credentials_file: Option<PathBuf>,
+    baker_credentials_file: Option<PathBuf>,
+    #[structopt(
+        long = "validator-credentials-file",
+        help = "Path to the validator credentials file. If the path is relative it is interpreted \
+                relative to the node process' working directory.",
+        env = "CONCORDIUM_NODE_VALIDATOR_CREDENTIALS_FILE"
+    )]
+    validator_credentials_file: Option<PathBuf>,
```

Comment:
I think it would be better to use the aliasing functionality i.e.
```suggestion
    #[structopt(
        long = "validator-credentials-file",
        alias = "baker-credentials-file",
        help = "Path to the validator credentials file. If the path is relative it is interpreted \
                relative to the node process' working directory.",
        env = "CONCORDIUM_NODE_VALIDATOR_CREDENTIALS_FILE"
    )]
    validator_credentials_file: Option<PathBuf>,
```
This doesn't support aliasing the environment variable, but we can just check `CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE` if this option is not set in `read_baker_credentials`. (Similarly for `decrypt-validator-credentials`.)

Reply:
I first thought that, but I think it's clearer that all environment variables are set in one place rather than randomly read somewhere in the code.